lib/project_types/script/layers/infrastructure/metadata_repository.rb

```diff
@@ -0,0 +1,18 @@
+
+module Script
+  module Layers
+    module Infrastructure
+      class MetadataRepository
+        include SmartProperties
+        property! :ctx, accepts: ShopifyCLI::Context
+
+        def get_metadata(file_location = "metadata.json")
```

Comment:
It might a good idea to force callers to pass in the metadata location, so that there's never any ambiguity about which file they're reading from.